config/initializers/devise.rb

```diff
@@ -168,7 +168,7 @@
   # :time  = Re-enables login after a certain amount of time (see :unlock_in below)
   # :both  = Enables both strategies
   # :none  = No unlock strategy. You should handle unlocking by yourself.
-  config.unlock_strategy = :time
+  config.unlock_strategy = :both
```

Comment:
I think from a business perspective, the only way we want someone to be able to unlock is if an admin unlocks them from this panel, and I think this might actually interfere with 'failed login attempts', which results in like a 15 minute 'chill out' ban.

This might be a thing where really what we're looking for is a `disabled` field rather than `locked`, since `locked` seems to mean something in the context of devise. (If that's the case let's do it in a separate PR and just leave the 'lock account' button nonfunctional for now)